en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,474 @@
+# Get Started with Choreo
+
+Choreo is a full lifecycle cloud-native developer platform that enables your developers to create, deploy and run new digital components like APIs, microservices, and automations in serverless mode on any Kubernetes cluster.
+
+This guide walks you through the following:
+
+- Developing, deploying, testing, and publishing a REST API.
+- How an external application consumes the published REST API.
+- How to release new versions of the REST API with added functionality.
+
+Let's consider a use case where a developer named Jane designs an application for reading lists. A Choreo developer named John develops a REST API for this application to consume. The two developers will develop two versions of the solution in two iterations. The second version of the application will add data persistence as a new feature.
+
+The following diagram summarizes the use case:
+
+![Use case summary](../assets/img/get-started/use-case-summary.png){.cInlineImage-full}
+
+Let's get started!
+
+## Prerequisites
+
+Before you try out this guide, complete the following:
+
+1. Create a GitHub repository to save the REST API implementation. For this guide, you can fork [https://github.com/Asitha/choreo-reading-list-application-example](https://github.com/Asitha/choreo-reading-list-application-example).
+
+2. If you are logging in to Choreo Console for the first time, create an organization as follows:
+
+    1. In [https://wso2.com/choreo/](https://wso2.com/choreo/), click **Try Choreo Now**.
+
+    2. Sign in to Choreo using your Google, GitHub, or Microsoft account.
+
+    3. Enter a unique organization name. For example, `Stark Industries`.
+
+        ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
+
+    4. Access and read the Privacy Policy and Terms of Use via the provided links, and select the **I agree with the Privacy Policy and Terms of Use** checkbox if you agree with them.
+
+    5. Click **Create** to add the new organization.
+
+    6. When the **Welcome to Choreo!** message appears, click either **Decline** or **Confirm** as you prefer depending on whether you want Choreo to send you emails with product updates, resources, and event invites.
+
+    You will be viewing the Home page of the Choreo Console.
+
+3. If required, create a new project as follows:
+
+    !!! info
+        If you do not create a new project, you can create the REST API for the default project.
+    
+    1. On the Home page of the Choreo Console, click **+ Create Project**.
+
+    2. Enter a unique name and a description for the project.
+
+    3. Click **Create**.
+
+## Step 1: Create and publish a REST API
+
+In this step, you are playing the role of John, the API developer. You will create and publish the REST API that the application developed by Sam needs to consume.
```

Comment:
who's Sam?

Reply:
The application developer. Updated the previous paragraph that mentioned a different name.